lib/matplotlib/rcsetup.py

```diff
@@ -549,7 +549,14 @@ def validate_sketch(s):
     try:
         return tuple(_listify_validator(validate_float, n=3)(s))
     except ValueError:
-        raise ValueError("Expected a (scale, length, randomness) triplet")
+        try:
+            result = tuple(_listify_validator(validate_float, n=4)(s))
+            # make sure seed is an integer
+            return (result[0], result[1], result[2], int(result[3]))
```

Comment:
As mentioned in :
https://github.com/matplotlib/matplotlib/issues/13479#issuecomment-466458114

We have to make sure seed is valid (int format) before it is passed on

Reply:
If a user passes a float instead of an int for the seed parameter, rather than raise an error, it just force casts it into an int and uses it . This is what I have seen after using a float seed value.

Exactly why it is made to force cast and use rather than raise an error is something I do not know. This section was worked on by the other 2 authors.